integration-tests/cli/csv-parser-price.it.js

```diff
@@ -111,10 +111,12 @@ describe('CSV and CLI Tests', () => {
       const csvFilePath = './packages/csv-parser-price/test/helpers/faulty-sample.csv'
 
       exec(`${binPath} -p ${projectKey} -i ${csvFilePath} --logLevel verbose`,
-        (error, stdout, stderr) => {
+        (error, stdout) => {
           expect(error.code).toBe(1)
-          expect(stdout).toBeFalsy()
-          expect(stderr).toMatch(/\.js:\d+:\d+/)
+          expect(stdout).toMatch('{"prices":[]}')
+
+          const errorLog = fs.readFileSync('csvparserprice.log', 'utf8')
```

Comment:
On other projects, I use multiple log streams which log everything into a log file and if there are any errors/criticals then they send logs also to stdout but it looks like the `npmlog` does support only one stream. I will play with it a little bit.